modules/microshift-embed-ostree-enable-eus-repos.adoc

```diff
@@ -0,0 +1,105 @@
+// Module included in the following assemblies:
+//
+// * microshift_install/microshift-embed-into-rpm-ostree.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-enable-eus-repos_{context}"]
+= Enabling extended support repositories for image building
+
+If you have an extended support (EUS) release of {microshift-short} or {op-system-base-full}, you must enable the {op-system-base} EUS repositories for image builder to use. If you do not have an EUS version, you can skip these steps.
+
+.Prerequisites
+
+* You have an EUS version of {microshift-short} or are updating to one.
```

Comment:
In #88190, this prereq also mentions `{op-system-base}`. Just noting that in case the prereqs here also need to mention that product.